src/api/flash/DFUUtilFlasher.js

```diff
@@ -21,70 +21,68 @@ import path from "path";
 
-export const DFUUtilFlasher = async (board, port, filename, options) => {
-  const callback = options
-    ? options.callback
-    : function () {
-        return;
-      };
-  const device = options.device;
-
-  const formatID = (desc) => {
-    return desc.vendorId.toString(16) + ":" + desc.productId.toString(16);
-  };
-
+const runDFUUtil = async (args) => {
   const dfuUtil = path.join(
     getStaticPath(),
     "dfu-util",
     `${process.platform}-${process.arch}`,
     "dfu-util"
   );
 
-  let dyld_library_path = "";
-  // dfu-util on darwin needs to know where its custom libusb is
-  if (process.platform === "darwin") {
-    dyld_library_path = path.join(
-      getStaticPath(),
-      "dfu-util",
-      process.platform
-    );
-  }
-
-  const runCommand = async (args) => {
-    const timeout = 1000 * 60 * 5;
-    return new Promise((resolve, reject) => {
-      logger("flash").debug("running dfu-util", { args: args });
-      const child = spawn(dfuUtil, args, {
-        env: { ...process.env, DYLD_LIBRARY_PATH: dyld_library_path },
-      });
-      child.stdout.on("data", (data) => {
-        logger("flash").debug("dfu-util:stdout:", { data: data.toString() });
-      });
-      child.stderr.on("data", (data) => {
-        logger("flash").debug("dfu-util:stderr:", { data: data.toString() });
-      });
-      const timer = setTimeout(() => {
-        child.kill();
-        logger("flash").error("dfu-util timed out");
-        reject("dfu-util timed out");
-      }, timeout);
-      child.on("exit", (code) => {
-        clearTimeout(timer);
-        if (code == 0 || code == 251) {
-          logger("flash").debug("dfu-util done");
-          resolve();
-        } else {
-          logger("flash").error("dfu-util exited abnormally", {
-            exitCode: code,
-          });
-          reject("dfu-util exited abnormally with an error code of " + code);
-        }
-      });
+  const timeout = 1000 * 60 * 5;
+  return new Promise((resolve, reject) => {
+    logger("flash").debug("running dfu-util", { args: args });
+    const child = spawn(dfuUtil, args);
+    child.stdout.on("data", (data) => {
+      logger("flash").debug("dfu-util:stdout:", { data: data.toString() });
+    });
+    child.stderr.on("data", (data) => {
+      logger("flash").debug("dfu-util:stderr:", { data: data.toString() });
+    });
+    const timer = setTimeout(() => {
+      child.kill();
+      logger("flash").error("dfu-util timed out");
+      reject("dfu-util timed out");
+    }, timeout);
+    child.on("exit", (code) => {
+      clearTimeout(timer);
+      if (code == 0 || code == 251) {
+        logger("flash").debug("dfu-util done");
+        resolve();
+      } else {
+        logger("flash").error("dfu-util exited abnormally", {
+          exitCode: code,
+        });
+        reject("dfu-util exited abnormally with an error code of " + code);
+      }
     });
-  };
+  });
+};
+
+const formatID = (desc) => {
```

Comment:
is that a USB Id? can we name it to be more descriptive? also, since other things are Id, calling this ID feels wrong.